central/delegatedregistryconfig/service/service_test.go

```diff
@@ -65,39 +79,39 @@ func TestGetClustersSuccess(t *testing.T) {
 
 	clustersDS := clusterDSMocks.NewMockDataStore(gomock.NewController(t))
 	deleClusterDS := deleDSMocks.NewMockDataStore(gomock.NewController(t))
-	s := New(deleClusterDS, clustersDS)
+	connMgr := connMgrMocks.NewMockManager(gomock.NewController(t))
 
-	genClusters := func(healthStatus *storage.ClusterHealthStatus) []*storage.Cluster {
-		return []*storage.Cluster{{Id: "id", Name: "fake", HealthStatus: healthStatus}}
-	}
+	s := New(deleClusterDS, clustersDS, connMgr)
+
+	clusters := []*storage.Cluster{{Id: "id", Name: "fake"}}
 
-	tt := []struct {
-		name     string
-		clusters []*storage.Cluster
-		valid    bool
+	tt := map[string]struct {
+		conn  *fakeSensorConn
+		valid bool
 	}{
-		{"missing health", genClusters(nil), false},
-		{"empty health", genClusters(clusterHealthStatusEmpty), false},
-		{"degraded", genClusters(clusterHealthStatusScannerDegraded), false},
-		{"healthy", genClusters(clusterHealthStatusScannerHealthy), true}, // only healthy scanners are valid
+		"without cap": {fakeConnWithoutCap, false},
+		"with cap":    {fakeConnWithCap, true}, // only healthy scanners are valid
```

Comment:
I don't think this comment is true anymore

Reply:
that code comment is not true, but your comment is true :) - nice catch